### DISCUSSION TITLE
How to disconnect dagster from slack organization

### DISCUSSION THREAD

**Discussion by spartakbughdaryan** [dagster-io/dagster#19671] [Q&A] [answered]
I added a slack organization in dagster cloud, and now i cant disconnect, how to disconnect it? I need to add a different organization
![image](https://github.com/dagster-io/dagster/assets/145757682/3d41f170-51d2-4aa1-bfad-69edf4018683)

**Comment by garethbrickman** [comment, marked as answer]
The Dagster Cloud Slack app will need to be removed from your Slack workspace. A workspace admin can do that by going to the app's Configuration and selecting the `Remove App` button:

![Screenshot 2024-02-08 at 11 14 36](https://github.com/dagster-io/dagster/assets/50636644/085e2580-1788-4f9a-a5f9-ff5afa5af889)

After this, refresh the Alerts page in Dagster Cloud and the button will now show the `Connect to Slack` option again.
